src/main/scala/com/typesafe/sbt/web/SbtWeb.scala

```diff
@@ -318,16 +298,34 @@ object SbtWeb extends AutoPlugin {
     nodeModules := nodeModuleGenerators(_.join).map(_.flatten).value
   )
 
+  def webJarsPathPrefix: Def.Initialize[Task[String]] = Def.task {
+    path(s"$WEBJARS_PATH_PREFIX/${moduleName.value}/${version.value}/")
+  }
+
+  def syncExportedAssets(): Def.Initialize[Task[File]] = Def.task {
+    val syncTargetDir = classDirectory.value
+    val syncRequired = TrackLevel.intersection(trackInternalDependencies.value, exportToInternal.value) match {
```

Comment:
I'm actually doing the same intersection call that you're doing in that diff... I'm confused as to how this should be changed.